docs/extensions/get-started/anatomy.md

```diff
@@ -65,17 +66,43 @@ Lens uses this ID to uniquely identify your extension.
     "react-open-doodles": "^1.0.5"
   },
   "devDependencies": {
-    "@k8slens/extensions": "^4.0.0-alpha.2",
+    "@k8slens/extensions": "^5.4.6",
     "ts-loader": "^8.0.4",
-    "typescript": "^4.0.3",
-    "@types/react": "^16.9.35",
-    "@types/node": "^12.0.0",
+    "typescript": "^4.5.5",
+    "@types/react": "^17.0.44",
+    "@types/node": "^14.18.12",
     "webpack": "^4.44.2",
     "webpack-cli": "^3.3.11"
   }
 }
 ```
 
+## Webpack configuation
+
+The following webpack `externals` are provided by `Lens` and must be used (when available) to make sure that the versions of that are used are in sync.
+
+| Package            | webpack external syntax     | Lens versions | Available in Main | Available in Renderer |
+| ------------------ | --------------------------- | ------------- | ----------------- | --------------------- |
+| `mobx`             | `var global.Mobx`           | `>5.0.0`      | ✅                | ✅                    |
+| `mobx-react`       | `var global.MobxReact`      | `>5.0.0`      | ❌                | ✅                    |
+| `react`            | `var global.React`          | `>5.0.0`      | ❌                | ✅                    |
+| `react-router`     | `var global.ReactRouter`    | `>5.0.0`      | ❌                | ✅                    |
+| `react-router-dom` | `var global.ReactRouterDom` | `>5.0.0`      | ❌                | ✅                    |
+| `react-dom`        | `var global.ReactDOM`       | `>5.5.0`      | ❌                | ✅                    |
+
+What is exported is the whole of the packages as a `*` import (within typescript).
+
+For example, if you want to use `mobx` in your main half an extension you would add the following to your webpack configuration file:
+
+```
+  "externals": [
+    ...
+    {
+      "mobx": "var global.Mobx"
```

Comment:
`"mobx-react": "var global.MobxReact"` is available too (in renderer)

Reply:
Yes, that is what the table above is supposed to denote. Did you find the table less than explanatory? Because if so, I should improve it.

Reply:
I think it's better just add all avaible varibles, most of "developers" probably just copy-and-paste.

Reply:
Okay will do